--- libki_local.conf.example	2016-07-19 10:26:22.472792070 -0400
+++ libki_local.conf	2016-09-14 12:08:26.969247165 -0400
@@ -1,9 +1,9 @@
 <Model::DB>
     traits Caching
     <connect_info>
-        dsn   dbi:mysql:libki
-        user   USERNAME
-        password "PASSWORD"
+        dsn   dbi:mysql:database=libki;host=libki_mysql
+        user   libkidb
+        password "libkidb"
         auto_savepoint 1
         quote_names 1
     </connect_info>
